card-data/01-character-upbringing.yaml

```
@@ -0,0 +1,264 @@
+
+      You grew up in a highly populated center of trade and culture.
+      People with needs offered what they had, or sometimes more,
+      and you did it too.
+
+      You're familiar with art, commerce, and political power.
+      You've seen people from all over, and have heard
+      or can even speak multiple languages.
+
+      You've learned to approach situations via negotiation.
+    prompts:
+      - Make a deal or haggle over details
+      - Navigate a physical or social labyrinth
+      - Gauge someone's wants or needs
+  back:
+    <<: *tback
+    name: City
+    desc: >
+      //"...For the right price." -- Basler
+
+      Not everything can be reduced to a commercial transaction,
+      but city dwellers sometimes try anyway.
+      Some of what you've seen has made you numb.
+      Other times, you've decided rightly or wrongly
+      that you can't help, and have moved on anyway.
+    prompts:
+      - Compromise a principle or ideal for expedience
+      - Overcomplicate a plan or solution
+      - Look past pain or hurt rather than soothe it
+- front: &Country
+    <<: *tfront
+    name: Country
+    desc: >
+      //"Take a deep breath of that country air!" -- Tana
+
+      You grew up among farmers, ranchers, or other folk who lived
+      off the land. You might have lived in a tight-knit community,
+      or been miles away from the nearest neighbor.
+
+      You're familiar with the plants or animals of your region.
+      You're used to putting in an honest day's work.
+
+      You've learned to approach situations via effort.
+    prompts:
+      - Endure wearying labor, physical or emotional
+      - Devise practical solutions to hard problems
+      - Understand someone's unspoken hardships
+  back:
+    <<: *tback
+    name: Country
+    desc: >
+      //"Smells like cow breakfast" -- Basler
+
+      Not everyone appreciates the necessity of hard work.
+      But sometimes, just plowing through isn't the right answer either.
+      You're used to getting down in the mud,
+      but it's important to reach for the stars as well.
+    prompts:
+      - Condemn someone for shunning hard work
+      - Balk at complex explanations or solutions
+      - Sacrifice tomorrow's hopes for today's needs
+- front: &Frontier
+    <<: *tfront
+    name: Frontier
+    desc: >
+      //"The forest demands self-sufficiency" -- Woody
+
+      You grew up at the edges of civilization, whether a harsh
+      desert, a lonely mountain, a deep forest, or somewhere
+      equally remote. You learned to fend for yourself from
+      a young age.
+
+      You can hunt, trap, or otherwise obtain your basic necessities.
+      Your life is a careful balance of trust and caution.
+
+      You've learned to approach situations through resourcefulness.
+    prompts:
+      - Come well-prepared for harsh conditions
+      - Spot what you need in the environment
+      - Anticipate danger
```

```suggestion
      - Come well-prepared for harsh conditions
      - Find what you need in the environment
      - Anticipate danger
```
Spot is a better word than find, aesthetically, but it reads a limiting to me.